src/dispatch/static/dispatch/src/participant/ParticipantAvatarGroup.vue

```diff
@@ -0,0 +1,208 @@
+    >
+      <template #activator="{ props: menu }">
+        <v-btn variant="text" v-bind="menu">
+          <!-- Display Visible Participants -->
+          <div class="avatar-row">
+            <!-- Display +n Avatar -->
+            <div v-if="hiddenParticipants.length > 0" class="avatar-container">
+              <v-avatar size="20px" class="extra-avatar">
+                +{{ hiddenParticipants.length }}
+              </v-avatar>
+            </div>
+            <div
+              v-for="(participant, index) in visibleParticipants"
+              :key="index"
+              class="avatar-container"
+            >
+              <v-avatar
+                size="20px"
+                :style="{ background: getAvatarGradient(participant.individual.name) }"
+                v-on="on"
+              />
+            </div>
+          </div>
+        </v-btn>
+      </template>
+      <v-card min-width="255" class="rounded-lg dispatch-side-card">
+        <v-list lines="one">
+          <v-list-item
+            v-for="(participant, index) in filteredParticipants"
+            :key="index"
+            @click="selectParticipant(participant)"
+            @mouseover="hoveredParticipant = participant"
+            @mouseleave="hoveredParticipant = ''"
+            density="compact"
+            rounded="lg"
+            active-class="ma-4"
+          >
+            <template #prepend>
+              <v-avatar
+                class="mr-n2"
+                size="12px"
+                :style="{ background: getAvatarGradient(participant.individual.name) }"
+              />
+              <!-- <v-icon class="mr-n6 ml-n2" size="x-small" icon="mdi-account"></v-icon> -->
+            </template>
+            <v-list-item-title class="item-title-font">
+              {{ participant.individual.name }}
+            </v-list-item-title>
+          </v-list-item>
+        </v-list>
+      </v-card>
+    </v-menu>
+  </div>
+</template>
+
+<style scoped>
+.avatar-row {
+  display: flex;
+  align-items: center;
+  justify-content: start;
+  flex-direction: row-reverse;
+  position: relative;
+}
+
+.avatar-container {
+  border: 2px solid white; /* Add a border around the avatar */
+  border-radius: 50%; /* Make the border circular */
+  position: relative;
+  margin-right: -5px; /* Adjust this value to change the overlapping amount */
+}
+
+.extra-avatar {
+  background-color: white;
+  color: rgb(60, 65, 73);
+  font-size: 10px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.hotkey {
+  vertical-align: baseline;
+  text-transform: capitalize;
+  text-align: center;
+  color: rgba(0, 0, 0, 0.816);
+  font-size: 11px;
+  line-height: 110%;
+  border-radius: 4px;
+  padding-left: 4px;
+  padding-right: 4px;
+  padding-top: 1px;
+  min-width: 17px;
+  border: 0.5px solid rgb(216, 216, 216);
+  background-color: rgb(254, 255, 254);
+  box-shadow: rgba(0, 0, 0, 0.086) 0px 2px 0px 0px;
+}
+
+.dispatch-side-card {
```

Comment:
I think I've seen this defined a couple of times, we should move it and consolidate.